### ISSUE TITLE
Final Project Review

### ISSUE THREAD

**Issue by louis-stp** [s0ap/orie5741-project#11]
This project intends to forecast the volatility of the market by using time series data of various sources, including SP500 data, indexes for asian markets, volatility indicators, and more. It looks at using conventional approaches in Financial Engineering such as GARCH models, but also investigates approaches reviewed in class. The conclusion is that their method performs better than a standard GARCH model, but is still very early and development and should not be relied upon in a production environment.

Pros:
I think this project was reasonable in their assessment in the difficulty of the problem and respected the complexity of the problem. They took several steps to address this such as splitting up testing vs training data in a way appropriate for time series, but most importantly I think they were very appropriate in discussing the value of their solution. It would be easy to look at positive results and say that it is great and perfect for deployment, but I like how they reiterated that it was still in development and still needed work.
The graphs they made were very high quality and easily interpretable.
The formatting and presentation of the document was very professional and well thought-out.

Cons:
It is not good that this team went 8 minutes over the limit for their presentation, and went 1 page over the limit for the report. It does not convey that they respect the guidelines of this project.
I think that this is a very advanced area, where special care needs to be taken to make sure every step you take is easily interpretable and also simply communicated. I am familiar with GARCH models and basic Financial Engineering concepts, but even I was lost and confused halfway through the report and presentation. If your goal is to communicate effectively with a general audience, or even an audience who understands data analysis but not financial engineering, I think you have failed this specific task.
Why did you not have a validation set? You can't select a model based off of the results of the test set, since that constitutes fitting based on the results of your test set, which is bad and not allowed! Use a validation set for this.
